src/styles/Popup_modal.scss

```diff
@@ -0,0 +1,191 @@
+    display: flex;
+    flex-direction: column;
+    overflow-y: scroll;
+    width: 450px;
+    height: 50%;
+    background-color: #3d3d4c;
+    padding: 20px;
+    border-radius: 10px;
+
+    button.modal-close-button {
+      position: fixed;
+      align-self: flex-end;
+      background-color: #ff000080;
+      color: #ffffff;
+      border-radius: 32px;
+      padding: 3px 8px;
+      border: none;
+      font-weight: 700;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ff0000;
+      }
+    }
+    .user-post-name {
+      font-weight: bold;
+      text-decoration: none;
+      color: #ffffff;
+      font-size: 1.2rem;
+    }
+    .user-post-body {
+      color: #ffffff;
+      word-wrap: break-word;
+      font-weight: 300;
+    }
+    .comment-card {
+      padding-bottom: 1rem;
+      border-bottom: 1px solid #ffffffb3;
+      .comment-user-name {
+        margin-top: 1rem;
+        font-weight: bold;
+        text-decoration: none;
+        color: #ffffff;
+        font-size: 1rem;
+      }
+      .comment-body {
+        color: #ffffff;
+        word-wrap: break-word;
+        font-weight: 300;
+      }
+      .comment-date {
+        font-size: 0.8rem;
+        color: #ffffff4d;
+      }
+    }
+
+    form.new-comment {
+      position: fixed;
+      top: auto;
+      bottom: 18%;
+      display: flex;
+      margin-bottom: 1rem;
+      .new-comment-input {
+        width: 300px;
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
+        color: #ffffff;
+        background-color: #48485a;
+        border-radius: 32px;
+        font-size: 1rem;
+      }
+      .new-comment-button {
+        margin-left: 1rem;
+        background-color: #50a0cd;
+        color: #ffffff;
+        border-radius: 32px;
+        padding: 3px 16px;
+        border: none;
+        font-weight: 700;
+
+        &:hover {
+          cursor: pointer;
+          background-color: #7fc3eb;
+        }
+        &:disabled {
+          color: #ffffff4d;
+          background-color: #37586b;
+        }
+      }
+    }
+  }
+}
+.modal-edit-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .modal-edit-content {
+    display: flex;
+    flex-direction: column;
+    width: 450px;
+    height: 50%;
+    background-color: #3d3d4c;
+    padding: 20px;
+    border-radius: 10px;
+
+    button.modal-edit-close-button {
+      position: fixed;
+      align-self: flex-end;
+      background-color: #ff000080;
+      color: #ffffff;
+      border-radius: 32px;
+      padding: 3px 8px;
+      border: none;
+      font-weight: 700;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ff0000;
+      }
+    }
+    .edit-post-header {
+      font-weight: bold;
+      text-decoration: none;
+      color: #ffffff;
+      margin-bottom: 1rem;
+    }
+    .edit-comment {
```

Comment:
Shouldn't it be `.edit-post`? ;-)

Reply:
changed names to match edit post instead of comment